Makefile

```diff
@@ -29,14 +29,19 @@ setupgolangcilint:  ## Install golangci-lint
 	@echo "==> Installing golangci-lint..."
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_VERSION)
 
+.PHONY: setupgokart
+setupgokart:  ## Install gokart
+	@echo "==> Installing gokart..."
+	go install github.com/praetorian-inc/gokart@latest
```

Comment:
This PR risks becoming more complex than probably you planned to but, installing latest could break evg by updating the linter to a newer version than what we may be running locally, that said go has a "feature" where doing go install with a pined version will pollute the `go.mod` file, we used to have this issue and we hacked it around with some `pushd`/`popd`, we can leave latest for now until it becomes a problem

Reply:
We seem to be doing it in other places too, e.g.: `check-licenses.sh`, which is why I went with `latest` and didn't fix the version in place.

Reply:
check licenses is more deterministic, it checks the file includes a license header but like I mention for now we can live with latest, I may give a try adding gokart as an official lint to golangci lint

Reply:
I found out that there isn't a way to ignore in `gokart`. That makes the tool less useful since we can't make it fail in CI (or locally).  One option would be to remove the use of `os.Environ()` in `mongosh.go`, but we may have a legitimate use for passing the env (now or in the future), making it difficult to work with.